README.md

```diff
@@ -47,3 +47,18 @@ foreach($parts['IMAGE/PNG'] as $image) {
     file_put_contents($image->name, $image->content);
 }
 ```
+
+## Content IDs
+
+Some emails embed images into the html content using "cid:" urls; these URLs link to the content ID of another part within the email rather than being a URL that can be resolved normally.
```

Comment:
```suggestion
Some emails embed images into the HTML content using `cid:` urls. These URLs link to the content ID of another part within the email, rather than being an absolute `https://` URL that can be resolved normally.
```